app/assets/stylesheets/preact/onboarding-modal.scss

```diff
@@ -319,15 +319,17 @@
       z-index: 2;
     }
 
-    &:hover {
+    &:hover,
+    &:focus-within {
```

Comment:
It has to be `:focus-within` instead of `:focus` because parent element that the focused button belongs to needs to be change transparency.

Reply:
Nothing! Awesome 😀